REFERENCES.md

```diff
@@ -0,0 +1,144 @@
+## Table of content
+
+- [2CP: Decentralized Protocols to Transparently Evaluate Contributivity in Blockchain Federated Learning Environments](#2CP-Decentralized-Protocols-to-Transparently-Evaluate-Contributivity-in-Blockchain-Federated-Learning-Environments)
+- [CAPC Learning: Confidential and Private Collaborative Learning](#CAPC-Learning-Confidential-and-Private-Collaborative-Learning)
+- [Distributionally Robust Federated Averaging](#Distributionally-Robust-Federated-Averaging)
+- [Towards Efficient Data Valuation Based on the Shapley Value](#Towards Efficient Data Valuation Based on the Shapley Value)
+_ [Transparent Contribution Evaluation for Secure Federated Learning on Blockchain](#Transparent Contribution Evaluation for Secure Federated Learning on Blockchain)
+- [Profit Allocation for Federated Learning](#Profit Allocation for Federated Learning)
+- [Transparent Contribution Evaluation for Secure Federated Learning on Blockchain](#Transparent Contribution Evaluation for Secure Federated Learning on Blockchain)
```

Comment:
Some links do not work

Reply:
it's still the case, starting from Towards Efficient Data Valuation Based on the Shapley Value